integration/feature/init/src/MillInitMavenTests.scala

```diff
@@ -0,0 +1,81 @@
+package mill.integration
+
+import mill.testkit.UtestIntegrationTestSuite
+import utest._
+
+object MillInitMavenTests extends UtestIntegrationTestSuite {
+
+  private def downloadExtractTo(workspace: os.Path, zipUrl: String, zipContentDir: String): Unit = {
+    val zipFile = os.temp(requests.get(zipUrl))
+    val contentDir = os.unzip(zipFile, os.temp.dir()) / zipContentDir
+    os.list(contentDir).foreach(os.move.into(_, workspace))
```

Comment:
It's a subjective style thing, but let's make most of these `foreach` blocks into `for` loops